.circleci/config.yml

```diff
@@ -42,9 +42,12 @@ jobs:
           command: |-
             mkdir -p /home/node/.npm-global
             ./.circleci/npm-install-retry.js
+            cd functions
+            ../.circleci/npm-install-retry.js
+            cd ..
           environment:
             NPM_CONFIG_PREFIX: /home/node/.npm-global
-      - run: cp defaultsettings.json settings.json
+      - run: cp functions/settings.tmpl.json functions/settings.json
```

Comment:
There's still a `defaultsettings.json` file sitting at the root of the repo that I'm assuming should also be renamed to `settings.tmpl.json`? In which case, the change to `config.yml` should actually be:

```
- run: cp settings.tmpl.json settings.json
- run: cp functions/settings.tmpl.json functions/settings.json
```

Reply:
The settings file in root is no longer being used so I just deleted defaultsettings.json and settings.json. CicleCi still fails.